### PR TITLE
fix: override of 'build.cssCodeSplit' will not take effect if `config.build.cssCodeSplit` is `undefined`

### PR DESCRIPTION
**Pull request by voldikss**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -42,18 +42,12 @@ export default function cssInjectedByJsPlugin({
             enforce: 'post',
             name: 'vite-plugin-css-injected-by-js',
             config(config, env) {
-                if (env.command === 'build') {
-                    if (!config.build) {
-                        config.build = {};
-                    }
-
-                    if (relativeCSSInjection == true) {
-                        if (config.build.cssCodeSplit == false) {
```
Comment (marco-prontera) [inline src/index.ts:51]: To fix what the title says it's enough to change this to: `if (!config.build.cssCodeSplit) {`

File: src/index.ts [hunk 2]
```diff
@@ -42,18 +42,12 @@ export default function cssInjectedByJsPlugin({
...
-                            config.build.cssCodeSplit = true;
-                            warnLog(
-                                `[vite-plugin-css-injected-by-js] Override of 'build.cssCodeSplit' option to true, it must be true when 'relativeCSSInjection' is enabled.`
-                            );
-                        }
+                if (env.command === 'serve') return
```
Comment (marco-prontera) [inline src/index.ts:45]: This is not necessary as the modification must only be made if the command is 'build'

File: src/index.ts [hunk 3]
```diff
@@ -42,18 +42,12 @@ export default function cssInjectedByJsPlugin({
...
+                if (!relativeCSSInjection) return
+                warnLog(`[vite-plugin-css-injected-by-js] Override of 'build.cssCodeSplit' option to true, it must be true when 'relativeCSSInjection' is enabled.`);
+                return {
+                    build: {
+                        cssCodeSplit: true
```
Comment (marco-prontera) [inline src/index.ts:50]: `return` isn't required

**Review by marco-prontera — CHANGES_REQUESTED** [review]
Thanks for your contribution! I think it is enough to change the code currently present in the repo with the first comment of this review and then we can proceed with the merge.

**Comment by voldikss** [comment]
@marco-prontera PTAL

**Comment by voldikss** [comment]
Hi @marco-prontera could this PR get merged?

**Review by marco-prontera — APPROVED** [review]

**Pull request merged**